modules/graphman-bundle.js

```diff
@@ -1,7 +1,7 @@
 
 const utils = require("./graphman-utils");
 const SCHEMA_METADATA = require("./graphman").schemaMetadata();
-const GOID_PLURAL_METHODS = ["fipUsers", "internalUsers", "fipGroups", "internalGroups", "ldaps", "fips", "trustedCerts"];
+const GOID_PLURAL_METHODS = ["federatedUsers", "internalUsers", "federatedGroups", "internalGroups", "ldapIdps", "federatedIdps", "trustedCerts"];
```

Comment:
I think, we should be supporting the deprecated entity types as part of dynamic query.

Reply:
added deprecated entity types